superset-frontend/src/explore/exploreUtils/getHostName.test.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { getHostName } from '.';
+
+jest.mock('src/utils/hostNamesConfig', () => ({
+  availableDomains: [
+    'domain-a',
+    'domain-b',
+    'domain-c',
+    'domain-d',
+    'domain-e',
+    'domain-f',
+    'domain-g',
+  ],
+}));
+
+test('Should get next diferent domain on a loop, never the first on the list', () => {
+  // eslint-disable-next-line no-plusplus
```

Comment:
I think we can remove the eslint exception and just implement the correct loop https://eslint.org/docs/rules/no-plusplus